nodejs/week1/warmup/circles.js

```diff
@@ -0,0 +1,25 @@
+
+class Circle {
+
+  constructor(radius) {
+    this.radius = radius;
+  }
+  getDiameter() {
+    return 2 * this.radius;
+  }
+  getCircumference() {
+    return 2 * Math.PI * this.radius;
+  }
+  getArea() {
+    return Math.PI * this.radius * this.radius;
```

Comment:
alternatively try using Math.pow(this.radius, 2) there's a bunch of useful functions in Math that come in handy when doing arithmetic operations

Reply:
i have included Math.pow function